src/card.scss

```diff
@@ -0,0 +1,124 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: $card-width;
+  padding: 32 16 16;
+  border-radius: 10px;
+
+  background: $main-white;
+  border: 1px solid $card-border-color;
+
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+
+    margin-bottom: 40;
+  }
+
+  &__description {
+    padding: 0;
+    margin: 0 0 4;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $link-color;
+
+    &__:hover {
+      color: #34568b;
+      transition: color 0.3s;
+    }
+  }
+
+  &__product-code {
+    margin: 0 0 16;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $secondary-accent;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24;
+
+    &-count {
+      margin: 0;
+
+      font-size: 10px;
+      font-weight: 400;
+      color: $secondary-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16;
+
+    color: $secondary-accent;
+
+    &-text {
+      margin: 0;
+
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: $link-color;
+    }
+
+    &-num {
+      margin: 0;
+      text-align: right;
+
+      height: 18px;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $link-color;
+    }
+  }
+
+  &__buy-button {
+    display: inline-block;
+    width: 100%;
+    height: 40px;
+
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    border-radius: 5px;
+    line-height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    color: $main-white;
+    background: $blue-accent;
+
+    transition: 0.3s;
+  }
+
+  &__buy-button:hover {
+    font-weight: 700;
+    color: $blue-accent;
+
+    background-color: $main-white;
+    border: 1px solid $blue-accent;
```

Comment:
- Use nesting in `scss`
- The font-weight doesn't change on hover
- It's better to change only the border color on hover instead of adding the whole border